vcpkg/maintainers/functions/vcpkg_make_install.md

```diff
@@ -0,0 +1,103 @@
+---
+title: vcpkg_make_install
+description: Use vcpkg_make_install to build and install a Make-based project.
+ms.date: 01/31/2024
+---
+# vcpkg_make_install
+
+Build and install a Make-based project.
+
+## Usage
+
+```cmake
+vcpkg_make_install(
+    [DISABLE_PARALLEL]
+    [DISABLE_DESTDIR]
+    [DISABLE_MSVC_FLAG_ESCAPING]
+    [USE_RESPONSE_FILES]
+    [LOGFILE_ROOT <root-name>]
+    [SUBPATH <sub-directory>]
+    [MAKEFILE <makefile-name>]
+    [TARGETS <target-name>...]
+    [OPTIONS <make-option>...]
+    [OPTIONS_RELEASE <make-option>...]
+    [OPTIONS_DEBUG <make-option>...]
+)
+```
+
+To use this function, you must depend on the helper port `vcpkg-make`:
+
+```json
+"dependencies": [
+  {
+    "name": "vcpkg-make",
+    "host": true
+  }
+]
+```
+
+## Parameters
+
+### ADD_BIN_TO_PATH
+
+Adds the configure dependent `(debug/)bin` directory to the system path. This is useful if configure builds and runs executables with vcpkg integrated dependencies.
+
+### DISABLE_PARALLEL
+
+By default, `vcpkg_make_install` will run make with the `-j` option to enable parallel building. If your project does not support parallel builds or you encounter issues with it, set this flag to disable parallel building. This will cause make to be executed without the -j option, running build steps sequentially.
+
+### DISABLE_DESTDIR
+
+By default, `make install` will use the `DESTDIR` variable to redirect installation to the package directory. Use this flag if `DESTDIR` should not be set.
+
+### DISABLE_MSVC_FLAG_ESCAPING
```

Comment:
Should this refer to the same-named option in `vcpkg_make_configure`?

What happens if they don't match (enabled in configure, disabled here, or vice-versa)?